package.json

```diff
@@ -149,7 +149,7 @@
     "less": "2.7.1",
     "less-loader": "2.2.3",
     "lodash": "3.10.1",
-    "marked": "0.3.6",
+    "markdown-it": "8.3.2",
```

Comment:
I'm +1 on enabling auto linking, for what it's worth.  If someone is putting a raw link into a markdown document, it's hard to imagine the circumstance where they want people to copy and paste that URL rather than just click it.

If we're happier with the markdown-it number approach, then we should pull the bandaid off.  This is is a major version change anyway, so breaking changes can happen.